WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabViewModel.cs

```diff
@@ -153,6 +152,11 @@ public ReceiveTabViewModel(WalletViewModel walletViewModel)
 				}
 			});
 
+			SaveQRCodeCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				await SelectedAddress?.SaveQRCodeAsync();
```

Comment:
This nullcheck doesn't work as far as I know it means if `SelectedAddress` is null then it's an `await null` which will fail with nullreferenceexception.